client/less/main.less

```diff
@@ -822,6 +822,11 @@ code {
       .btn-danger {
         background: #FFFFFF;
       }
+      .btn-success:hover,
```

Comment:
Is this commit in the wrong branch?

Reply:
@Bouncey WIP. 
No, I'm keeping everything related to settings **(night mode)** in this PR. Actually after 2 hours of constant hardwork `npm run develop` worked. `node seed` works fine but db doesn't get connected on running `npm run develop`. I tried many fixes, edited mongoHQ in .env(later copied sample.env to .env), checked firewall, reinstalled mongo, for me getting connected to db is a chance of luck :(. 
So, I'm doing max changes in one go. I hope you'll understand. 

OS: Windows 10 Pro 32 Bit.

Reply:
Do you any errors in your `mongod` terminal?

Reply:
@Bouncey No there's no error in mongod. Here's what I get in my `npm run develop` console: 

```shell
fcc:server connecting to db at mongodb://localhost:27017/freecodecamp +1ms
Connection fails: MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connection 0 to localhost:27017 timed out]
It will be retried for the next request.

D:\Projects & Repositories\01. Repository - freeCodeCamp\freeCodeCamp\node_modules\mongodb\lib\mongo_client.js:421
          throw err
          ^
MongoError: failed to connect to server [localhost:27017] on first connect [MongoError: connection 1 to localhost:27017 timed out]
    at Pool.<anonymous> (D:\Projects & Repositories\01. Repository - freeCodeCamp\freeCodeCamp\node_modules\mongodb-core\lib\topologies\server.js:336:35)
    at emitOne (events.js:116:13)
    at Pool.emit (events.js:211:7)
    at Connection.<anonymous> (D:\Projects & Repositories\01. Repository - freeCodeCamp\freeCodeCamp\node_modules\mongodb-core\lib\connection\pool.js:280:12)
    at Object.onceWrapper (events.js:317:30)
    at emitTwo (events.js:126:13)
    at Connection.emit (events.js:214:7)
    at Socket.<anonymous> (D:\Projects & Repositories\01. Repository - freeCodeCamp\freeCodeCamp\node_modules\mongodb-core\lib\connection\connection.js:197:10)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:106:13)
    at Socket.emit (events.js:208:7)
    at Socket._onTimeout (net.js:407:8)
    at ontimeout (timers.js:475:11)
    at tryOnTimeout (timers.js:310:5)
    at Timer.listOnTimeout (timers.js:270:5)
[23:16:01] [nodemon] app crashed - waiting for file changes before starting...
rs
```